### PR TITLE
Adding Next & Mui : Theming + Initial Folder Structure

### PR DESCRIPTION
**Pull request by EricLiclair**
### Fixes #<number>
<Do not delete this template>
Description: <provide a description> The title should be short and relevant.

#### Checklist
- [x] I've followed the Contributing guidelines provided in the repository.
- [x] I've made the changes which were demanded in the linked issue.
- [x] I've tested my code on a Chromium based browser.
- [x] I've tested my code on Mozilla Firefox.
- [x] My code gave a clean console on debugging. (no warnings/errors)

#### Testimonial
<provide a relevant screenshot or any testimonial for the solved issue if applicable>

Initial home
image

Folder structure
image

Ignorable warning for Lottie file: will be removed once development starts.
image

### CODE REVIEW

File: client/.gitignore [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
```
Comment (EmperorYP7) [inline client/.gitignore]: this file should be deleted

**Review by EmperorYP7 — CHANGES_REQUESTED** [review]
Apart from that, everything LGTM. Well done @ericLiclair

**Comment by SrijanSriv** [comment]
@EricLiclair while you're at it with the folder restructuring, sort the multiple `Code_of_Conduct.md` thing as well. Also, theming includes only mui right?

**Review by EmperorYP7 — APPROVED** [review]
LGTM

**Comment by EricLiclair** [comment]
Hey, @EmperorYP7 thanks for the changes. Also, @SrijanSriv Yes. Theming is purely done using themprovider @mui . Thought of adding framer but i believe that can be added when implementing animations.

**Pull request merged**